cluster/stone-prd-rh01/tenants/hcc-pipeline-tenant/storage_broker_tekton_test.yaml

```diff
@@ -24,7 +24,11 @@ spec:
         value: ingress
       - name: COMPONENT_NAME #IMPORTANT: Your component in RHTAP has to be named the same as this field.
         value: storage-broker
+      - name: IQE_PLUGINS
+        value:
```

Comment:
Dalia can you please provide the empty string here? I believe it would be safer.